Sources_v3/StreamChat/ChatClient.swift

```diff
@@ -69,19 +69,15 @@ public typealias ChatClient = _ChatClient<DefaultExtraData>
 ///
 public class _ChatClient<ExtraData: ExtraDataTypes> {
     /// The `UserId` of the currently logged in user.
-    @Atomic public var currentUserId: UserId = .anonymous
-    
+    @Atomic public var currentUserId: UserId?
```

Comment:
Why don't we / are we going to:
- Parse the last logged user id from DB (I think we're going to do this anyway)
- If there's no last logged in user, currentUserId is `anonymous`
so that we don't have to deal with optional here?

Reply:
I think this approach is more correct and @evsaev's explanation of this in the description makes sense. I like the change.

Reply:
Please take a look at the `Making the current user ID optional` section in the PR description, I gave some reasoning for making current user id non-optional.